packages/core/src/basebackend.ts

```diff
@@ -93,6 +96,20 @@ export abstract class BaseBackend<O extends Options> implements Backend {
     });
   }
 
+  /**
+   * @inheritDoc
+   */
+  public sendSession(session: Session): void {
+    if (!this._transport.sendSession) {
+      logger.warn('sendSession not implemented for a transport used');
```

Comment:
Our SDK-provided transports all must implement `sendSession` -- we can make the message more clear and drive people to what happened and where the problem is:

```suggestion
      logger.warn('Dropping session because custom transport doesn't implement sendSession');
```

When I see this warning,

1. I know a session was dropped, nothing else
2. I know it is because I use a custom transport, I need to go see what is this `sendSession` that is not implemented.

---

(for a separate issue...)

_What happened with transactions, though? Our SDK-provided transports know how to handle the "transaction events", but most likely custom transports will not. Would have been probably cleaner to not expect custom transports to send transactions._

Reply:
Updated. 

As for Transactions, I'd say it's fine, because they are opt-in and regular error events that go through pipe will work just fine with all "older custom transports". Where here, I couldn't change `sendEvent` interface, as it'd break the contract.